.github/workflows/autotag.yml

```diff
@@ -0,0 +1,30 @@
+name: Auto Tag
+
+on:
+  schedule:
+    - cron: "1 12 * * 5"
```

Comment:
Looks good to me, but why the `1` there instead of `0`? Any particular reason?

Reply:
I always went into the habit of not doing round numbers for scheduled stuff due to everyone doing so. So there are usually spikes of activity around the top of the hour on CI infra where you can do cronjobs

Reply:
Sounds good! Thanks.